### PR TITLE
General code cleanup.

### PR DESCRIPTION
**Pull request by rpmoore**

### CODE REVIEW

File: ds3-sdk/src/main/java/com/spectralogic/ds3client/helpers/Ds3ClientHelpers.java [hunk 1]
```diff
@@ -290,8 +290,7 @@ public static Ds3ClientHelpers wrap(final Ds3Client client, final int retryAfter
      *                   response.isTruncated() is true. To return the additional keys, see nextMarker.
      * @throws IOException
```
Comment (DenverM80) [inline ds3-sdk/src/main/java/com/spectralogic/ds3client/helpers/Ds3ClientHelpers.java]: update comment

File: ds3-sdk/src/main/java/com/spectralogic/ds3client/helpers/Ds3ClientHelpers.java [hunk 2]
```diff
@@ -312,7 +311,7 @@ public static Ds3ClientHelpers wrap(final Ds3Client client, final int retryAfter
      * @param retries    Specifies how many times the helper function will attempt to retry a request for failing. Default - 5
      * @throws IOException
      */
-    public abstract Iterable<Contents> listObjects(final String bucket, final String keyPrefix, final String nextMarker, final int maxKeys, final int retries) throws IOException;
```
Comment (DenverM80) [inline ds3-sdk/src/main/java/com/spectralogic/ds3client/helpers/Ds3ClientHelpers.java]: update comment

File: ds3-sdk/src/main/java/com/spectralogic/ds3client/utils/collections/LazyIterable.java [hunk 3]
```diff
@@ -62,15 +62,15 @@ public LazyIterable(final LazyLoaderFactory<T> lazyLoaderFactory) {
         List<T> getNextValues();
     }
 
-    private class LazyObjectIterator<T> implements Iterator<T> {
+    private class LazyObjectIterator<E> implements Iterator<E> {
```
Comment (RachelTucker) [inline ds3-sdk/src/main/java/com/spectralogic/ds3client/utils/collections/LazyIterable.java]: Why was this changed from T to E?

Comment (rpmoore) [inline ds3-sdk/src/main/java/com/spectralogic/ds3client/utils/collections/LazyIterable.java]: It was complaining about the type being hidden.  My assumption is that it didn't like that we already had a type T defined in the parent class.
